.github/workflows/ci.yaml

```diff
@@ -6,4 +6,6 @@ on:
 
 jobs:
   ci:
-    uses: nextstrain/.github/.github/workflows/pathogen-repo-ci.yaml@master
+    uses: nextstrain/.github/.github/workflows/pathogen-repo-ci.yaml@dec0880059017dac7facf100435c5737bf1386c8
+    with:
+      workflow-root: phylogenetic
```

Comment:
This workflow does not fully work as intended (see [Tom's comment in original PR](https://github.com/nextstrain/.github/pull/57#discussion_r1336389847)).

You may want to use the stopgap I've used in mpox (https://github.com/nextstrain/mpox/pull/214) until the pathogen-repo-ci is updated.

Reply:
Thanks for the explanation! Incorporated changes in commit https://github.com/nextstrain/dengue/pull/15/commits/79d1be45d3917f8350d25585beb2eeff625a3cd4